message/validation/validation_test.go

```diff
@@ -37,6 +37,11 @@ import (
 	"github.com/bloxapp/ssv/utils/rsaencryption"
 )
 
+// taken from network/commons/common.go
+const (
+	messageOffset = 264
```

Comment:
Should we make it exported in `network/commons/common.go` instead?

Reply:
Yep, agree. Made it exported